cmake/toolchain/zephyr/generic.cmake

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-if(NOT DEFINED SDK_VERSION)
-  message(FATAL_ERROR "SDK_VERSION must be set")
+if (NOT ZEPHYR_SDK_INSTALL_DIR)
+  message(FATAL_ERROR "ZEPHYR_SDK_INSTALL_DIR must be set")
 endif()
```

Comment:
Why not have both?

Are you certain it's not possible for SDK_VERSION to be unset while ZEPHYR_SDK_INSTALL_DIR is set?

Reply:
The sdk version is read from the sdk_version file in the ZEPHYR_SDK_INSTALL_DIR path. The file is checked for existence, hence if ZEPHYR_SDK_INSTALL_DIR is correct then SDK_VERSION will be set.
All this is checked in cmake/toolchain/zephyr/host-tools.cmake